### DISCUSSION TITLE
sympy.physics.mechanics keyword acceleration_constraints

### DISCUSSION THREAD

**Discussion by Peter230655** [sympy/sympy#27194] [Q&A]
My question is mostly curiosity:
The documentation says:
![image](https://github.com/user-attachments/assets/8772f939-8a7b-40b2-87f7-de47bcc0b374)
In my simulations, these acceleration constraints have always been velocity_constraints.diff(t) - in which case they need not be given.
I cannot think of an example, where acceleration_constraints have an 'independent' use.
I'd be **thankful** for an example to broaden my ideas of stuff to simulate..

**Comment by moorepants** [comment]
I think it was in place if you wanted to ensure that all q dots and q double dots were replaced, as that wasn't done automatically in the past. You may be able to construct these in a way that makes simpler expressions than simply calling `velcon.diff(t)`. But, it could probably be deprecated and then removed.

**Comment by Peter230655** [comment]
**Thanks!** 
My question has been answered, so I will close this.
(Since it does not hurt, maybe one can also leave it for backward compatibility?)
